### PR TITLE
Bug Fix: Document the helper scripts

### PR DESCRIPTION
**Pull request by BenjamenMeyer**
Add some documentation about the helper scripts

Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [x] This is a documentation change only

Issues:
- https://github.com/vegastrike/Assets-Masters/issues/23

Purpose:
Document the helper scripts that @evertvorster created.

### CODE REVIEW

**Review by Loki1950 — APPROVED** [review]

**Comment by stephengtuggy** [comment]
So I'm not actually sure that the `vsettings` script or icon has made it into the assets beta yet. I think they may have been committed on master only. Let's make sure that that is rectified, otherwise we will be documenting a script that doesn't exist.

File: README.md [hunk 1]
```diff
@@ -326,6 +326,23 @@ make package
 Gameplay
 ========
 
+Running Vega Strike
+-------------------
+
+Starting with 0.7.x there are some helper scripts for running Vega Strike and it's configuration utility:
```
Comment (stephengtuggy) [inline README.md:332]: `its` not `it's`

**Review by stephengtuggy — CHANGES_REQUESTED** [review]
One minor change to request. Otherwise, looks good.

**Comment by BenjamenMeyer** [comment]
I checked the 0.7.x Beta 3 Deb file and can confirm the scripts are in there.

File: README.md (hunk 1 above)
Comment (BenjamenMeyer) [reply README.md]: fixed in latest commit

**Review by stephengtuggy — APPROVED** [review]
Looks good to me!

**Pull request merged**